doc/question.md

```diff
@@ -110,6 +88,78 @@ The `question.html` is regular HTML, with four special features:
 
 4. LaTeX equations are available within HTML by using `$x^2$` for inline equations, and `$$x^2$$` or `\[x^2\]` for display equations.
 
+## Question `server.py`
```

Comment:
Moved the `server.py` section below the `question.html` section since the `server.py` skeleton includes a `question.html`.